src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -270,5 +270,10 @@ private void OnMembersListBoxMouseWheel(object sender, MouseWheelEventArgs e)
             // do not propagate to child items with scrollable content
             e.Handled = true;
         }
+
+        private void OnMoreInfoClicked(object sender, RoutedEventArgs e)
+        {
+            ViewModel.dynamoViewModel.OpenDocumentationLinkCommand.Execute(new OpenDocumentationLinkEventArgs(new Uri(Dynamo.Wpf.Properties.Resources.NodeAutocompleteDocumentationUriString, UriKind.Relative)));
```

Comment:
is there a pattern you are following for adding this to the resx file?

I hope this does not get localized - that would cause problems wouldn't it?

Reply:
yes it's done the same way for Python editor documentation and publish package doc as well

Reply:
@QilongTang it might be worth checking with localization team if this is going to cause a problem - if the url is localized then this link won't work in localized environments.

Reply:
@mjkkirschner Thanks, I have reached out to localization team. https://autodesk.slack.com/archives/C01FVCEMYTA/p1621415985002200